string fan_name
uint32 incubator_id
